### PR TITLE
Fix a memory leak in the bullpen view

### PR DESCRIPTION
**Pull request by jpmartin2**
#### What issue does this fix?
- Memory leak in BullpenView
#### What features does this propose?
- Removes the `values` list in the BullpenView, which just kept references to PieceViews, which was never cleared, leaking memory.

### CODE REVIEW

**Comment by PeterMitrano** [comment]
You also removed the timer....

**Comment by jpmartin2** [comment]
The timer that only prints a message. The timer that actually updates the view is in the view (and not removed).

**Comment by PeterMitrano** [comment]
Ok my bad

**Comment by PeterMitrano** [comment]
You also removed the timer....

**Comment by PeterMitrano** [comment]
:+1:

**Comment by jkuszmaul** [comment]
But that timer also updates `elapsedTime`

**Comment by jpmartin2** [comment]
Ah... Right.

**Comment by jpmartin2** [comment]
Do we want to keep both timers? It seems kind of odd to have two. Maybe have the level keep track of elapsed time, and have the view register as a listener or something?

**Comment by jkuszmaul** [comment]
Moving it to just two would be better. I'm not sure which to move out.

**Comment by jpmartin2** [comment]
So I got it working again, with just one timer. Another alternative to what I've done here would be to have a controller that has the timer, so the model doesn't have to know about the view.

**Comment by PeterMitrano** [comment]
I feel like that's preferable then model having view but I haven't looked
at it so just my 2c

**Comment by jpmartin2** [comment]
Yeah, right now I reused the RefreshListener interface (so the model has an instance of that), but I think having a controller might be preferable - though I'd have to think a bit about how to deal w/ different types of levels.

File: src/main/java/aeneas/views/PlayLevelView.java [hunk 1]
```diff
@@ -65,6 +62,7 @@
 
   PlayLevelView(Level levelModel, Model model, MainView mainView) {
     this.levelModel = levelModel;
+    levelModel.setListener(this);
```
Comment (PeterMitrano) [inline src/main/java/aeneas/views/PlayLevelView.java]: rename this method from `setListener` to `setRefreshListener`. Otherwise I'll plus one it

**Pull request merged**